languages/julia/config.json

```
@@ -21,6 +21,20 @@
                 "concepts": ["boolean-logic", "booleans"],
                 "prerequisites": ["functions-introduction"]
             },
+            {
+                "uuid": "36cbda04-3921-4491-9ec6-76951f00aa00",
+                "slug": "matrices-introduction",
+                "concepts": [
+                    "matrices-introduction",
```

To me, the `-introduction` suffix doesn't add much and could perhaps be replaced with just `matrices`:

```suggestion
                    "matrices",
```

As in: this concept introduces matrices. If you feel like this is a good suggestion, the exercise itself could also be renamed.

---

I want to mark explicitely that it's the first exercise about matrices, but there will be more. When I read `matrices` for me would be the set containing all `

---

Okay. We can always change this later,

---

I think this should be `introduction-to-matrices` in that case.

---

that looks weird in the list